metadata.json

```diff
@@ -26,7 +26,8 @@
     {
       "operatingsystem": "Debian",
       "operatingsystemrelease": [
-        "9"
+        "9",
+        "10"
```

Comment:
Shouldn't we add Centos to this list too?

Reply:
CentOS is already there, for release version 7.